assets/shaders/custom_material.wgsl

```diff
@@ -13,5 +15,8 @@ var base_color_sampler: sampler;
 fn fragment(
     #import bevy_pbr::mesh_vertex_output
 ) -> @location(0) vec4<f32> {
-    return material.color * textureSample(base_color_texture, base_color_sampler, uv);
+    var time = (sin(globals.time) + 1.0) / 2.0;
```

Comment:
This can be a `let`. Also, there's an operation available in basically all GPUs called 'fused multiply-add' that can do multiplication and addition in one instruction that if I recall correctly is both faster and more accurate. I'm not sure how good shader compilers are at figuring out that something could be a fused multiply-add, but I've tried to write things in that way since I read: https://momentsingraphics.de/FMA.html
```suggestion
    let time = sin(globals.time) * 0.5 + 0.5;
```

Reply:
Woah, good to know didn't know about that.